### ISSUE TITLE
`pip install pokemaster` failure

### ISSUE THREAD

**Issue by kipyin** [kipyin/pokemaster#10]
`pip install pokemaster` does not work:

```
$ pip install pokemaster
...
Collecting pokedex (from pokemaster)
  Could not find a version that satisfies the requirement pokedex (from pokemaster) (from versions: )
No matching distribution found for pokedex (from pokemaster)
...
```

I've searched all existing issues on `poetry` and cannot find a solution to this (or even a relevant issue). I think `pokedex` should be shipped with `pokemaster`.

**Comment by kipyin** [comment]
The issue with shipping `pokedex` with `pokemaster`: 

1. Too many files (all the csv data).
2. Getting and setting the default database will require extra work (exponentially).

So, nah. I think I'll try to fix this issue on the PyPI side.

**Comment by kipyin** [comment]
Note: `pip install git+https://github.com/veekun/pokedex` works. How should I put this into `pyproject.toml`?

**Comment by kipyin** [comment]
So this is actually an issue with `poetry`, who strips the git dependencies when building the package. I can:

1. Wait for a fix.
2. Make PR to `poetry`. But I have no idea how to fix the wheel part.
3. Use `requirements.txt` for now.

**Comment by kipyin** [comment]
I think this is solved with ` [7c4845b](/kipyin/pokemaster/commit/7c4845be8fdb02832961c1aaa71c742730f80e23)` and ` [fd9453b](/kipyin/pokemaster/commit/fd9453b202a90f5859bee5afc83aecfa935b76db)`.
